source/docs/installation.md

```diff
@@ -79,5 +75,21 @@ At the time of writing this guide, the output looked like this:
 
 <pre><code class="shell">3.2.1</code></pre>
 
+## Installing Embark @next
+
+While running `npm install -g embark` will give us the latest and great stable version of Embark, we're sometimes interested in trying out features that are in active development. The Embark maintains a `next` distribution tag on npm that can be used to install versions of Embark that aren't stable.
+
+<pre><code class="shell">$ npm -g install embark@next</code></pre>
+
+## Installing Embark's latest nightly
+
+If you're interested in getting whatever has landed last in the code base, we can go ahead and install the latest version of Embark's GitHub repository like this:
+
+<pre><code class="shell">$ npm -g install github.com/embark-framework/embark</code></pre>
```

Comment:
Re: the *latest nightly* section:

I don't think it's correct to refer to it as a *nightly*, since the develop branch could change at any time, multiple times, on any given day, while it may not change at all on the weekends.

Maybe change the section header to *Installing from GitHub*.

So how about:

```
If you're interested in trying out the latest changes to Embark, you can install directly
from the `develop` branch (the default branch) on GitHub:

<pre><code class="shell">$ npm -g install embark-framework/embark</code></pre>
```